app/controllers/passwords_controller.rb

```diff
@@ -32,11 +32,11 @@ def update
   private
 
   def require_valid_token
-    user = User.find_by(reset_token: params[:token])
+    user = User.find_by(token: params[:token])
 
-    unless user.present? && params[:token].present? && user.reset_token_expiration >= DateTime.now
-      render 'invalid_token'
+    unless user.present? && params[:token].present? && user.token_expiration >= DateTime.now
```

Comment:
`user.token_expiration >= DateTime.now` this could be moved to model as `user.expired_token?`